ui/src/overridden/backoffice/Importer/ImportedDocuments.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Accordion, Button, Header, Icon, Message } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { http } from '@inveniosoftware/react-invenio-app-ils';
+import _get from 'lodash/get';
+import { ReportDetails } from './ReportDetails';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import { DocumentIcon } from '@inveniosoftware/react-invenio-app-ils';
+
+const importerURL = '/importer/check';
+
+export class ImportedDocuments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: -1,
+      importCompleted: false,
+      data: null,
+      intervalId: null,
+      isLoading: true,
+    };
+  }
+
+  componentDidMount() {
+    //TODO: Define how much we wait until the next call (currently 30 sec)
```

Comment:
TODO warning. Why not using a config instead?
30 secs looks a lot to me... Then I would expect to see something like: checking in 30... 29... etc...